docs/specs/resolve.yml

```diff
@@ -109,9 +109,10 @@ outputs:
     { "content": "<p><a target=\"_blank\" href=\"c\">c</a></p>" }
   docs/dir/c.json:
 ---
-# Show warning if the link is empty, missing or points to absolute file path
+# Show warning if the link is empty, missing
 inputs:
-  docfx.yml:
+  docfx.yml: |
+    exclude: '*'
```

Comment:
>exclude: '*' [](start = 4, length = 12)

this actually change the user behavior, they just need specific which folder need to be build and publish at first, but right now i believe most of the repos need exclude "*"(root level files) in their config

Reply:
and also if you made this change,  configuration migrator tool need to be changed also.

---
In reply to: [245192658](https://github.com/dotnet/docfx/pull/3860#discussion_r245192658) [](ancestors = 245192658)